roles/openshift_aws/defaults/main.yml

```diff
@@ -286,6 +284,8 @@ openshift_aws_vpc:
     - cidr: 172.31.16.0/20
       az: "us-east-1a"
 
+openshift_aws_subnet_az: "{{ openshift_aws_vpc.subnets[openshift_aws_region][0].az }}"
```

Comment:
@joelddiaz,

I'm not sure how I feel about this.  One one hand I see the simplicity of using the vpc because the az is defined there.  

On the other hand I'm not sure I like using an array element for the variable in case others (team member, ansible, some tool like yaml that sorts) change, move, or unexpectedly update the openshift_aws_vpc variable data order.

Thoughts?

Reply:
I'm just working through provisioning to something besides us-east-1, and thought it would be nice to have one less variable that I have to define/override in that case. After defining my own openshift_aws_vpc structure for region us-east-2, the provisioning playbook was erroring out in non-obvious ways because I hadn't set openshift_aws_subnet_name.

Do you think it's worthwhile to add an example openshift_aws_vpc variable in the provisioning_vars.yml.example right above the reference to openshift_aws_subnet_name? This might make it more clear that doing a non-default openshfit_aws_vpc means you should also update openshift_aws_subnet_name.